sources/PTYTask.m

```diff
@@ -43,6 +43,8 @@
                 struct winsize* win,
                 int width,
                 int height,
+                int pixelWidth,
```

Comment:
Shall I apply this to the existing width/height params? They aren't used inside these functions besides to assign ws_row/col.
Same regarding `launchWithPath` and `setWidth:...`